manager/integration/tests/test_settings.py

```diff
@@ -228,4 +231,98 @@ def test_setting_guaranteed_engine_cpu():
 
     Note: use fixture to restore the setting into the original state
     """
-    pass
+    client = get_longhorn_api_client()
+    core_api = get_core_api_client()
+
+    # Get current guaranteed engine cpu setting and save it to org_setting
+    setting = client.by_id_setting(SETTING_GUARANTEED_ENGINE_CPU)
+    org_setting = setting
+
+    # Get current running instance managers
+    instance_managers = client.list_instance_manager()
+
+    # Set an invalid value, and it should return error
+    try:
+        client.update(setting, value="xxx")
+    except Exception as e:
+        print("Exception when update "
```

Comment:
We should expect an exception. See e.g. https://github.com/longhorn/longhorn-tests/blob/master/manager/integration/tests/test_basic.py#L109